infra/embedding_step_functions/unified_embedding/handlers/compaction.py

```
@@ -917,9 +1135,88 @@ def download_and_merge_delta(
+        except Exception:
+            pass
+
+        # Get validation duration if available
+        validation_duration = 0.0
+        try:
+            if 'validation_duration' in locals():
+                validation_duration = validation_duration
+            else:
+                validation_duration = time.time() - validation_start_time
+        except Exception:
+            pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)